hafnian/__init__.py

```diff
@@ -47,6 +47,9 @@
     matrices with non-negative elements. This is done by sampling determinants;
     the larger the number of samples taken, the higher the accuracy.
 
+Batched hafnian algorithm
+    TODO
```

Comment:
@nquesada or @bgupt, realised this definition was missing, could one of you guys fill it in? This is in the code API section, so just needs a short 1-2 sentence of what it does plus a link to the paper.

Something like
```rest
Returns photon number statistics of a Gaussian state for a given
covariance matrix as described in
*Multidimensional Hermite polynomials and photon distribution for polymode mixed light*
`arxiv:9308033 <https://arxiv.org/abs/hep-th/9308033>`__.
```

Reply:
Done!

Reply:
Thanks @nquesada!

Reply:
👍